index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>SMART ON FHIR STARTER APP</title>
+   
+   <script src="./lib/fhir-client.js"></script>
+    <script src='./load_data.js'></script>
+    <script src='./lib/jquery.min.js'></script>
+    
+  <script>
+      extractData().then(function(p){

```

Comment:
It will be nice to move the UI updates after the data is loaded to another file say : smart_starter_controller.js . 
This controller would then hold all the code to update the UI. 
This will also help us in keeping the HTML clean by keeping all the JS (for updating the UI) out in its own file.